deployment/constructs/lambda.ts

```diff
@@ -94,6 +109,15 @@ export class Lambda extends Construct {
           authorizationType: isLocalStack
             ? undefined
             : apigateway.AuthorizationType.IAM,
+          requestParameters: requestParameters
```

Comment:
Not sure what benefit we are really getting from requestParameters at all. Is it important of could it be left out in all apps without a problem?

Reply:
Discussed and we don't need request parameters in the construct